.changeset/serious-gifts-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/canon': patch
```

Comment:
Big set of breaking changes here so this should be `minor` and you'll want to prefix the message with `**BREAKING**:`